benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -268,11 +280,21 @@ public void invoke(Req request, io.grpc.stub.StreamObserver<Resp> responseObserv
     }
   }
 
-  private static final class BenchmarkServiceDescriptorSupplier implements io.grpc.protobuf.ProtoFileDescriptorSupplier {
+  private static final class BenchmarkServiceDescriptorSupplier implements io.grpc.protobuf.ProtoDescriptorSupplier {
     @java.lang.Override
     public com.google.protobuf.Descriptors.FileDescriptor getFileDescriptor() {
       return io.grpc.benchmarks.proto.Services.getDescriptor();
```

Comment:
This call is expensive, because it loads a lot of descriptor information that is not necessary much of the time. The ProtoFileDescriptorSupplier was specifically designed to avoid loading that information when unnecessary.

Instead, I'd suggest copying what we did for the ServiceDescriptor: just place the BenchmarkServiceDescriptorSupplier instance on the object, and let the caller call `getFileDescriptor().findServiceByName(blah).findMethodByName(blah)` manually.

We could provide utility methods in `ProtoUtils` to do the searching though, since given a `MethodDescriptor` we know what values to pass to the `find*` methods.